nft_ingester/src/bin/backfill/main.rs

```diff
@@ -62,16 +62,18 @@ async fn main() {
         Storage::open_readonly_with_cfs_only_db(&args.source_db_path, SlotStorage::cf_names())
             .expect("Failed to open source RocksDB");
     let red_metrics = Arc::new(RequestErrorDurationMetrics::new());
+    let cancellation_token = CancellationToken::new();
+    let stop_handle = tokio::task::spawn({
+        let cancellation_token = cancellation_token.clone();
+        async move {
+            usecase::graceful_stop::graceful_shutdown(cancellation_token).await;
```

Comment:
I believe it should be ok to set up those interruption handlers twice - here and below, but just double checking

Reply:
wdym?

Reply:
I mean we setup the interruption signal handler twice - in this method and directly in Ctrl+C handler slightly below. That should be ok, just was doublechecking

Reply:
yeah i think it's fine